Engine/UserInput.cs

```diff
@@ -318,16 +373,15 @@ bool isBrowsable(IMemberData m)
                         Console.Write($"({str.Value}): ");
                     else Console.WriteLine(":");
                     
-                    
                     if (secure)
                     {
-                        var read2 = (awaitReadLine(timeoutAt, true) ?? "").Trim();
+                        var read2 = (awaitReadLine(timeoutAt, true)).Trim();
```

Comment:
looks like awaitReadLine could previously return null. This is no longer the case?

Reply:
No, `awaitReadLine` was used in two places, and both places used null propagation, so I moved the null propagation into `awaitReadLine` instead